src/viam/api/CMakeLists.txt

```diff
@@ -178,12 +181,98 @@ if (VIAMCPPSDK_USE_DYNAMIC_PROTOS)
+    ${PROTO_GEN_DIR}/google/api/expr/v1beta1/expr.grpc.pb.h
+    ${PROTO_GEN_DIR}/google/api/expr/v1beta1/expr.pb.cc
+    ${PROTO_GEN_DIR}/google/api/expr/v1beta1/expr.pb.h
+    ${PROTO_GEN_DIR}/google/api/expr/v1beta1/source.grpc.pb.cc
+    ${PROTO_GEN_DIR}/google/api/expr/v1beta1/source.grpc.pb.h
+    ${PROTO_GEN_DIR}/google/api/expr/v1beta1/source.pb.cc
+    ${PROTO_GEN_DIR}/google/api/expr/v1beta1/source.pb.h
+    ${PROTO_GEN_DIR}/google/api/expr/v1beta1/value.grpc.pb.cc
+    ${PROTO_GEN_DIR}/google/api/expr/v1beta1/value.grpc.pb.h
+    ${PROTO_GEN_DIR}/google/api/expr/v1beta1/value.pb.cc
+    ${PROTO_GEN_DIR}/google/api/expr/v1beta1/value.pb.h
+    ${PROTO_GEN_DIR}/google/api/field_behavior.grpc.pb.cc
+    ${PROTO_GEN_DIR}/google/api/field_behavior.grpc.pb.h
+    ${PROTO_GEN_DIR}/google/api/field_behavior.pb.cc
+    ${PROTO_GEN_DIR}/google/api/field_behavior.pb.h
+    ${PROTO_GEN_DIR}/google/api/field_info.grpc.pb.cc
+    ${PROTO_GEN_DIR}/google/api/field_info.grpc.pb.h
+    ${PROTO_GEN_DIR}/google/api/field_info.pb.cc
+    ${PROTO_GEN_DIR}/google/api/field_info.pb.h
+    ${PROTO_GEN_DIR}/google/api/http.grpc.pb.cc
+    ${PROTO_GEN_DIR}/google/api/http.grpc.pb.h
     ${PROTO_GEN_DIR}/google/api/http.pb.cc
     ${PROTO_GEN_DIR}/google/api/http.pb.h
+    ${PROTO_GEN_DIR}/google/api/httpbody.grpc.pb.cc
+    ${PROTO_GEN_DIR}/google/api/httpbody.grpc.pb.h
+    ${PROTO_GEN_DIR}/google/api/httpbody.pb.cc
+    ${PROTO_GEN_DIR}/google/api/httpbody.pb.h
+    ${PROTO_GEN_DIR}/google/api/launch_stage.grpc.pb.cc
+    ${PROTO_GEN_DIR}/google/api/launch_stage.grpc.pb.h
+    ${PROTO_GEN_DIR}/google/api/launch_stage.pb.cc
+    ${PROTO_GEN_DIR}/google/api/launch_stage.pb.h
+    ${PROTO_GEN_DIR}/google/api/resource.grpc.pb.cc
+    ${PROTO_GEN_DIR}/google/api/resource.grpc.pb.h
+    ${PROTO_GEN_DIR}/google/api/resource.pb.cc
+    ${PROTO_GEN_DIR}/google/api/resource.pb.h
+    ${PROTO_GEN_DIR}/google/api/visibility.grpc.pb.cc
+    ${PROTO_GEN_DIR}/google/api/visibility.grpc.pb.h
+    ${PROTO_GEN_DIR}/google/api/visibility.pb.cc
+    ${PROTO_GEN_DIR}/google/api/visibility.pb.h
+    ${PROTO_GEN_DIR}/google/rpc/code.grpc.pb.cc
+    ${PROTO_GEN_DIR}/google/rpc/code.grpc.pb.h
+    ${PROTO_GEN_DIR}/google/rpc/code.pb.cc
+    ${PROTO_GEN_DIR}/google/rpc/code.pb.h
+    ${PROTO_GEN_DIR}/google/rpc/context/attribute_context.grpc.pb.cc
+    ${PROTO_GEN_DIR}/google/rpc/context/attribute_context.grpc.pb.h
+    ${PROTO_GEN_DIR}/google/rpc/context/attribute_context.pb.cc
+    ${PROTO_GEN_DIR}/google/rpc/context/attribute_context.pb.h
+    ${PROTO_GEN_DIR}/google/rpc/error_details.grpc.pb.cc
+    ${PROTO_GEN_DIR}/google/rpc/error_details.grpc.pb.h
+    ${PROTO_GEN_DIR}/google/rpc/error_details.pb.cc
+    ${PROTO_GEN_DIR}/google/rpc/error_details.pb.h
+    ${PROTO_GEN_DIR}/google/rpc/status.grpc.pb.cc
+    ${PROTO_GEN_DIR}/google/rpc/status.grpc.pb.h
+    ${PROTO_GEN_DIR}/google/rpc/status.pb.cc
+    ${PROTO_GEN_DIR}/google/rpc/status.pb.h
```

Comment:
This might be my mistake: I had assumed that all the generated grpc code needed to go in here, and this was all new since the last time the C++ SDK was synced with the API repo. I'll try pruning it down...

Reply:
You were totally right, thanks for pushing back on this!